index.html

```diff
@@ -41,5 +41,19 @@
     </a>
 </div>
 </div>
-    </body>
+
+<div class="ui divider"></div>
```

Comment:
I think in the future we can consider wrapping the content in a padded container instead of adding dividers between the content and the nav bar. For now this should be fine

Reply:
Wouldn't it be better to change it now instead of later so that we don't have to deal with it? @dimitraz

Reply:
Sure, I just didn't want you to change the scope of your PR, since you'd probably have to go back and change multiple pages. Maybe we can merge this and you can open up a new PR for that?